guacamole/src/main/java/org/apache/guacamole/tunnel/TunnelRequestService.java

```diff
@@ -226,7 +300,7 @@ protected GuacamoleTunnel createConnectedTunnel(UserContext context,
      * @throws GuacamoleException
      *     If an error occurs while obtaining the tunnel.
      */
-    protected GuacamoleTunnel createAssociatedTunnel(GuacamoleTunnel tunnel,
+    protected GuacamoleTunnel createAssociatedTunnel(final GuacamoleTunnel tunnel,
```

Comment:
Any particular reason for changing this to final, here?  Might be a good reason, just asking...